### PR TITLE
Feature/lol/move stream layer tool

### PR DESCRIPTION
**Pull request by lliss**
Connects #763

Provides a custom leaflet control to show and hide the stream selector.

To test:
 * Find the new map control and click it.
 * Move slider back and forth.
 * Ensure streams appear.

Notes:
 * Need a custom icon to match the mocks. Used some "streamy" looking arrows for the time being.
 * Controls jump upon opening. Could use some designer love to prevent this.

### CODE REVIEW

**Comment by lliss** [comment]
@azavea-bot rebuild

**Comment by caseypt** [comment]
Should opening the layer control or the stream slider close the other if that one is open? It seems like that needs to happen or we should have the control window stack horizontally.

**Comment by caseypt** [comment]
To have the control buttons stack (like the zoom in/out buttons), it looks like we have to wrap them all in a `div` with the class `leaflet-bar`. And then each control gets the class `leaflet-bar-part`. Consider doing taht here.

File: src/mmw/js/src/core/streamSliderControl.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        if (!L.Browser.touch) {
+            L.DomEvent
+                .disableClickPropagation(container)
+                .disableScrollPropagation(container);
+        } else {
+            L.DomEvent.on(container, 'click', L.DomEvent.stopPropagation);
+        }
+        this.streamFeatures.addTo(map);
+        return container;
+    },
```
Comment (caseypt) [inline src/mmw/js/src/core/streamSliderControl.js]: Missing space.

**Comment by caseypt** [comment]
+1. Test. Nice job.

@mmcfarland Can we get some designer time next sprint to help with this and the layer control?

**Comment by mmcfarland** [comment]
@caseypt Yep, however in the meantime we're going to just be hiding the streams (#778)

**Comment by caseypt** [comment]
To help with styling efforts, consider moving the locate control to the bottom right of the map. https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/core/views.js#L104

**Comment by lliss** [comment]
> To have the control buttons stack (like the zoom in/out buttons), it looks like we have to wrap them all in a div with the class leaflet-bar. And then each control gets the class leaflet-bar-part. Consider doing that here.

@casey Moving these around is actually pretty difficult considering how we're building the HTML for them. Were you able to make these look the right way on your end.

**Comment by caseypt** [comment]
@lliss I didn't realize there was a special classes for keeping the buttons together when I was building the layer control. I thought they would just stack if another control was added.

**Comment by lliss** [comment]
> @lliss I didn't realize there was a special classes for keeping the buttons together when I was building the layer control. I thought they would just stack if another control was added.

My preference is to punt on this and try to resolve all the style issues at once later on.

**Comment by caseypt** [comment]
Sounds good to me.

**Pull request merged**